index.bs

```diff
@@ -7418,12 +7418,12 @@ of the same shape.
 
 	1. If \(v\) is equal to zero, return -1000.
 
-	2. Else, return \( 20 \, \log_{10}{v} \)
+	2. Else, return \( 20 \, \log_{10}{v} \).
 </div>
 
 <div algorithm="convert db to linear">
 	Converting a value \(v\) in <dfn>decibels to
-	linear gain unit</dfn> means returning \(10^\frac{v}{20}\)
+	linear gain unit</dfn> means returning \(10^{v/20}\).
```

Comment:
The displayed fraction is really tiny and is also as tall as 10.  Don't use a fraction to make the exponent bigger and easier to read.